WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -253,6 +253,15 @@ private extension ProductsViewController {
         }
         coordinatingController.start()
     }
+
+    func clearFilter(sourceBarButtonItem: UIBarButtonItem? = nil, sourceView: UIView? = nil) {
+        ServiceLocator.analytics.track(.productListClearFiltersTapped)
+        self.filters = FilterProductListViewModel.Filters(stockStatus: nil,
```

Comment:
The `self` should not be necessary, it is implicit and we tend to not make it explicit in Swift (it makes the code a bit more concise).

Also, given all the properties in Filters are passed as nil, we could also rely on the initializer without parameters that Filters provides. So this line could be:

`filters = FilterProductListViewModel.Filters()`

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5372/commits/cbd9a6cdd0f0920421ad321ea37b141391e46d1c and https://github.com/woocommerce/woocommerce-ios/pull/5372/commits/6a3f93674522be07a79e57ccd49f3747bc76cb98